viewer/src/views/threejs/cad/MaterialManager.ts

```diff
@@ -25,29 +25,42 @@ function updateVisibility(visible: VisibilityDelegate, materials: Materials, tre
 export class MaterialManager {
   public readonly materials: Readonly<Materials>;
   private readonly _options?: NodeAppearance;
+  private _clippingPlanes: THREE.Plane[] = [];
+  private _clipIntersection: boolean = false;
 
   constructor(treeIndexCount: number, options?: NodeAppearance) {
     this.materials = createMaterials(treeIndexCount);
     this._options = options;
     this.setRenderMode(RenderMode.Color);
   }
 
+  get clippingPlanes(): THREE.Plane[] {
+    return this._clippingPlanes;
+  }
+
+  set clippingPlanes(clippingPlanes: THREE.Plane[]) {
+    this._clippingPlanes = clippingPlanes;
+    this.applyToAllMaterials(material => {
+      material.clippingPlanes = clippingPlanes;
+      console.log('Setting', material, clippingPlanes);
+    });
+  }
+
+  get clipIntersection(): boolean {
```

Comment:
It's a.bit confusing what intersection means. Is there a better name for it? Or maybr using a boolean enum?